do.py

```diff
@@ -101,30 +102,32 @@ def setup_ext():
 
 
 def setup():
-    run(
-        [
-            py() + " -m pip install --upgrade pip",
-            py() + " -m pip install --upgrade virtualenv",
-            py() + " -m virtualenv .env",
-        ]
-    )
+    if platform.system().lower() == "windows":
```

Comment:
do we need to check explicitly for windows ? isn't it true for all platforms ?

Reply:
yes, virtualenv module is supported only for python2.7 in windows and in python3 venv is build-in

Reply:
I mean we can remove the check for windows right ? venv should be available for all platforms for python3.6+.

Reply:
I doubt for linux

Reply:
https://docs.python.org/3/library/venv.html suggests it should be supported for all platforms.